package.json

```diff
@@ -42,7 +42,7 @@
     "doctrine": "^1.2.0",
     "es6-map": "^0.1.3",
     "escope": "^3.6.0",
-    "espree": "^3.1.2",
+    "espree": "git+https://github.com/eslint/espree.git",
```

Comment:
Will change to latest version of espree when it's released, but even running `npm install espree@next` only installed the latest tagged version, not the latest version from the repo. Wanted to make the PR with this version to get feedback.